### DISCUSSION TITLE
unexpected failure: externalStrippedSwfs\assets\.... .swf(The system cannot find path specified)

### DISCUSSION THREAD

**Discussion by rahimhaji** [airsdk/Adobe-Runtime-Support#877] [Q&A]
Dear friends,

iam creating an app with Adobe Air 33.1.1.476, during compile iam getting these attached error. I have valid licence from Harman. Previously i used to compile all the air version it was working fine and generating externalstrrippedswfs, but when i start doing its giving below error. Pls help me to solve this.

iam getting "unexpected failure: externalStrippedSwfs\assets\.... .swf(The system cannot find path specified) " error.

![image](https://user-images.githubusercontent.com/11812885/119771821-53e5aa00-becf-11eb-99d3-405a07cbfc20.png)

Please note that those folders & swf's are available in assets folder.

please help me to solve this. 

Thanks in advance,
Syed Abdul Rahim

**Comment by ajwfrost** [comment]
Hi
I'm wondering whether this is caused by a working folder location.. the trace above shows that the Java code is trying to create a new file:
```
externalStrippedSwfs\assets\...\filename.swf
```
and prior to the call into the `copyTo` utility, the code is doing a `mkdirs()` call that should cause all of the folder structures to be created properly so the file creation then works. It seems like the `mkdirs()` call is failing hence there is no path available when we try to create a file there.

If it's failing, it may be because the Java application has a working folder of somewhere that we don't have write permissions for...

As a short term fix, it might be that you can run Animate as an administrator which may mean the Java process also gets admin rights; it would then be interesting to see where the files are created! Or launch Animate from a command line that's set to a working folder where you have write access..

The generated folder should really be in the subfolder of where the IPA file would be created.. which means we maybe need to make it more explicit in the code to ensure the external stripped swfs are put somewhere as a subfolder of this - and/or we should add some more error handling so that you can see what's happening and where it's trying to write the files to.

thanks

**Reply by rahimhaji** [reply]
Dear Mr.Andrew,

Greetings! Thanks for your reply. Even i made my self in working directory externalStrippedSwfs folder then also i got same error.Please let me know how to set outputfolder during compile? any idea?

Thanks and Regards,
Syed Abdul Rahim

**Reply by ajwfrost** [reply]
If you're using Animate then I don't know how that would work.. if you can set up ADT to output some logging information then it might allow you to run ADT directly from a command-line, rather than using Animate.

For this, if you go into the SDK's "lib" folder and edit the `adt.cfg` file, add a line `DebugOut=true`
Then restart Animate and publish the app, and then look under `c:\users\username` for a file called `adt.log`.
This should have the command listed that was used by Animate, that you can then copy onto a command-line...

thanks

  Andrew

**Reply by rahimhaji** [reply]
Dear Mr.Andrew,

Greetings! i try to open animate as administrator, then also it gives same error as above (system cannot find path....). Any ways to solve this issues?

Thanks and Regards,
Syed Abdul Rahim

**Reply by rahimhaji** [reply]
Thanks Mr.Andrew,

its works fine after i opened as administrator.. thanks for the helpe.

**Comment by rahimhaji** [comment]
Dear Mr.Andrew&friends,

now iam getting id:can't buffer of 210637712 for output architecture armv7, error (refer below image) during adobe air compiling. Please note that already i increased my java heap size to 2048mb. Please help me to solve the issue.. its urgent for me...

![image](https://user-images.githubusercontent.com/11812885/121995559-2dbd7680-cdb8-11eb-8c62-d346e109e4dc.png)

Thanks and Regards,
Syed Abdul Rahim

**Reply by rahimhaji** [reply]
its coming in the end of the compile.
